src/main/java/graphql/schema2/GraphQLSchema.java

```diff
@@ -0,0 +1,31 @@
+package graphql.schema2;
+
+import graphql.schema.visibility.GraphqlFieldVisibility;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class GraphQLSchema {
```

Comment:
I think you need to put together "strawman" examples of how you see the DataFetchers and Schema getting wired together.

Is there a GraphqlSchemaBuilder that wires DFs and DatafetcherFactories to Fields?  By name, by direct field object?

Should this be merged some how with the current RuntimeWiring idea which is really this already?